### DISCUSSION TITLE
Feed & chart metatrader 4 data

### DISCUSSION THREAD

**Discussion by umarzuki** [tradingview/lightweight-charts#802] [General]
Hi,

It is possible to use mt4 data for lightweight charts on same server? If yes, any example/docs I can follow?

Thanks.

**Comment by timocov** [comment]
I think it should be definitely possible as soon as lightweight-charts doesn't restrict you from technologies you might use for your website and what datafeed you want to use.

We don't have any examples for that so far, but I think it shouldn't be hard to adapt any existent example of connecting mt4 with any another web solution to use with lightweight-charts.
